modules/ROOT/pages/_partials/rn-known-issues-ref.adoc

```diff
@@ -3,15 +3,42 @@
 
 This section lists known issues with  {PRODUCT} {MINOR_VERSION_NO}:
 
-* In the _Managing and protecting APIs using API management_ Solution Pattern, users must have administrator privileges to create a product in 3scale. Administrator privileges are not required for other Solution Patterns.
+// https://issues.redhat.com/browse/INTLY-9939
+* In Managed Integration SSO, setting the email details for the admin user within the *Manage account* page fails.
```

Comment:
in the "Manage account" or from the ... not within

Reply:
Changed to 'in'.